native/cocos/bindings/manual/jsb_global.cpp

```diff
@@ -92,11 +92,13 @@ static cc::network::Downloader *localDownloader() {
 }
 
 static void localDownloaderCreateTask(const ccstd::string &url, const std::function<void(const ccstd::string &, unsigned char *, int)> &callback) {
+#if CC_PLATFORM != CC_PLATFORM_OPENHARMONY // TODO(qgh):May be removed later
```

Comment:
It is better to describe the reason, then will not forget the reason in future.

Reply:
The network seems to be supported, I'll test it